### DISCUSSION TITLE
Central mp3 directory for panel radios

### DISCUSSION THREAD

**Discussion by beto-sp-1985** [AzuraCast/AzuraCast#3970] [Ideas]
Hi I'm from Brazil. Congratulations on the great Azuracast panel, I'm really enjoying it!

Would you like to see if I can create a directory that will be available for the radios selected in the panel?
Example: a directory with mp3 files as recorded programs and available for radios selected in the Azuracast panel.

How can I do this?

One suggestion: it would be integration with the WHMCS and having the broadcasters function to insert a certain time recorded with announcers.

Awaiting return. Thanks!

**Comment by beto-sp-1985** [comment]
* Correction: The suggestion is the right time recorded with announcers. The panel would have the function of searching for the right recorded time, so it would be able to inform the right time (recorded) as it is on fm radio.

Sorry, I don't know how to speak English, I use the Google translator, sometimes it may not be very well explained to my suggestions.

Thank you!

**Comment by silvaalex2021** [comment]
@beto-sp-1985 Eu não entendi mesmo o que você escreveu no último comentário. Mas Você já tentou usar o On demand do painel? É só ativar nas configurações da rádio. Eu não usei muito mas eu acho que pode ser o que você precisa.
